docs/mission.md

```diff
@@ -0,0 +1,21 @@
+# Knative Eventing Mission Statements
+
+## 10-word version
+
+Enable asynchronous application development through event delivery from anywhere.
+
+## 25-word version
+
+Knative/Eventing offers a consistent, Kubernetes-based platform for reliable, secure and scalable asynchronous event delivery from packaged or app-created event sources to on-cluster and off-cluster consumers.
+
+## Full Mission Statement
+
+The Knative Eventing mission focuses on three areas: Developer Experience, Event Delivery, and Event Sourcing. The primary focus is Developer experience around event driven applications; event delivery and sourcing provide infrastructure which enables the developer experience to scale to complex applications.
+
+By leveraging open standards and infrastructure (such as CloudEvents and Kubernetes), Knative Eventing provides a robust and extensible ecosystem which accommodates interoperability between open and closed-source frameworks and services.
+
+**Developer Experience:** Enable developers to write, maintain, and support event driven applications by providing a uniform experience when sourcing events from either their own systems and external systems.
+
+**Event Delivery:** Provide an implementation of event delivery infrastructure, accompanied by a well documented specification. Enable a pluggable architecture for the event delivery mechanism so that developers and operators can choose the correct event delivery substrate for their requirements.
```

Comment:
'documented specification'. I think we should also provide conformance tests to accompany the spec?

Reply:
added reference to conformance tests.